adapter/internal/interceptor/interceptor.go

```diff
@@ -119,20 +125,24 @@ local res_call_config = {
 	{{- end -}}}
 function envoy_on_response(response_handle)
 	interceptor.handle_response_interceptor(
-		response_handle, res_call_config, resp_flow_list
+		response_handle, res_call_config, resp_flow_list, wire_log_config
 	)
 end
 `
 	// defaultRequestInterceptorTemplate is the template that is applied when request flow is disabled
 	// just updated req flow info with  resp flow without calling interceptor service
 	defaultRequestInterceptorTemplate = `
 function envoy_on_request(request_handle)
-	interceptor.handle_request_interceptor(request_handle, {}, {}, resp_flow_list, inv_context, true)
+	utils.wire_log_body(request_handle, " >> request body >> ", {{ .LogConfig.LogBodyEnabled }})
```

Comment:
Shall we print headers first, then body and finally the trailiers?

Reply:
updated the log order to be headers -> body -> trailers